### PR TITLE
add schema to docs

### PR DESCRIPTION
**Pull request by m-kharbat**

### CODE REVIEW

File: blackbox/docs/sql/administration/user_defined_functions.txt [hunk 1]
```diff
@@ -127,7 +142,10 @@ This would overload our ``multiply`` function with more arguments::
 .. NOTE::
 
    It is considered bad practice to create functions that have the same name
-   as the CrateDB built-in functions.
+   as the CrateDB built-in functions. When no schema name is provided, the
```
Comment (bogensberger) [inline blackbox/docs/sql/administration/user_defined_functions.txt]: Even if it's slightly longer i would prefer, to have it more clear. How about this?

... built-in functions. If you call a function without a schema name, CrateDB will look up the builtin functions first and then the user-defined functions in the session schema. Therefore a built-in function with the same name as a user defined function will hide the latter. However, ...

File: blackbox/docs/sql/administration/user_defined_functions.txt [hunk 2]
```diff
@@ -159,6 +177,10 @@ Optionally, argument names can be specified within the drop statement::
      cr> DROP FUNCTION IF EXISTS calculate_distance(start_point geo_point, end_point geo_point);
      DROP OK, 1 row affected  (... sec)
 
+Optionally, if a function was created in a specific schema, it can be dropped like this::
```
Comment (bogensberger) [inline blackbox/docs/sql/administration/user_defined_functions.txt]: Optionally, you can provide a schema::

File: blackbox/docs/sql/administration/user_defined_functions.txt [hunk 3]
```diff
@@ -80,6 +80,21 @@ For example, if you wanted to name two ``geo_point`` arguments ``start_point`` a
    Argument names are used for query documentation purposes only. You cannot
    reference arguments by name in the function body.
 
+Optionally, a schema for the function can be specified. If no schema is
+specified, the function will be created in the current session schema.
+
```
Comment (bogensberger) [inline blackbox/docs/sql/administration/user_defined_functions.txt]: Optionally, you can specify a schema for the function. If you omit the schema, the function
is stored in the current session schema.

Comment (nslater) [inline blackbox/docs/sql/administration/user_defined_functions.txt]: Optionally, you can specify a schema for the function. If you omit the schema, the current session schema is used.

File: blackbox/docs/sql/administration/user_defined_functions.txt (hunk 1 above)
Comment (nslater) [inline blackbox/docs/sql/administration/user_defined_functions.txt]: editing @bogensberger's suggestion:

> ... built-in functions.
>
> If you call a function without a schema name, CrateDB will first look up the builtin functions, and then the user-defined functions in the session schema. Therefore, a built-in function with the same name as a user defined function will always hide the latte when called without a scheme name.
>
> However, user-defined functions with the same name as a built-in function can still be...

File: blackbox/docs/sql/administration/user_defined_functions.txt (hunk 2 above)
Comment (nslater) [inline blackbox/docs/sql/administration/user_defined_functions.txt]: I'm not sure I understand this suggestion?

**Pull request merged**